activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/model/IntegrationRequestEvent.java

```diff
@@ -31,18 +37,36 @@
 
     //used by json deserialization
     public IntegrationRequestEvent() {
+        this.id = UUID.randomUUID().toString();
+    }
+
+    public IntegrationRequestEvent(String id,
```

Comment:
Are you sure that we need this constructor? It seems to be never used by us, and the json deserialization is more likely going to use the default construct.

Reply:
@erdemedeiros true, let's remove it. But something important to notice is that I've added that ID to every event, we need to have it, even though the executionId should be enough.

Reply:
👍 
@Salaboy BTW I noticed that the message itself provides an ID as a header.